### PR TITLE
COM-18-Implement-Global-Error-Handling

### PR DESCRIPTION
**Pull request by valentyna-koshelnyk**

### CODE REVIEW

**Comment by MenilV** [comment]
This PR now has confilicts, let's resolve them first before reviewing.

**Comment by valentyna-koshelnyk** [comment]
> This PR now has confilicts, let's resolve them first before reviewing.

@MenilV resolved :)

File: src/main/java/com/startsteps/ecommerceapi/user/config/Swagger2Configuration.java
Comment (MenilV) [inline src/main/java/com/startsteps/ecommerceapi/user/config/Swagger2Configuration.java:1]: do we need this file now?

File: src/main/java/com/startsteps/ecommerceapi/user/security/jwt/JwtUtil.java [hunk 1]
```diff
@@ -76,12 +83,28 @@ public boolean validateJwtToken(String authToken) {
...
 
+//    public String generateTokenFromUsername(String username) {
+//        return Jwts.builder()
+//                .setSubject(username)
+//                .setIssuedAt(new Date())
+//                .setExpiration(new Date((new Date()).getTime() + jwtExpirationMs))
+//                .signWith(key(), SignatureAlgorithm.HS256)
+//                .compact();
+//    }
+
```
Comment (MenilV) [inline src/main/java/com/startsteps/ecommerceapi/user/security/jwt/JwtUtil.java]: do we need this?

**Review by MenilV — CHANGES_REQUESTED** [review]
A few things to check but good overall.

File: src/main/resources/application-dev.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ ecom:
     key: 5pAq6zRyX8bC3dV2wS7gN1mK9jF0hL4tUoP6iBvE3nG8xZaQrY7cW2fA # according to SHA-256 requirements
     algorithm: HS256
     issuer: http://localhost:8080
-    expiresIn: 30
+    expiresIn: 300000000
```
Comment (MenilV) [inline src/main/resources/application-dev.yml:6]: this seems like a lot (3.5 days)

**Review by MenilV — APPROVED** [review]
nice!

**Pull request merged**